### PR TITLE
fix(snapshot): support invariant entry parsing

### PR DESCRIPTION
**Pull request by mattsse**
## Motivation
parse invariant test entries in snapshots correctly

cc @transmissions11

### CODE REVIEW

File: cli/src/cmd/forge/snapshot.rs [hunk 1]
```diff
@@ -439,4 +455,18 @@ mod tests {
             }
         );
     }
+
+    #[test]
+    fn can_parse_invariant_snapshot_entry() {
+        let s = "Test:deposit() (runs: 256, calls: 100, reverts:200)";
```
Comment (onbjerg) [inline cli/src/cmd/forge/snapshot.rs:461]: is the missing space between reverts: and 200 part of the cli output too? if so should prolly just fix that

**Review by onbjerg — APPROVED** [review]
lgtm

File: cli/src/cmd/forge/snapshot.rs (hunk 1 above)
Comment (mattsse) [reply cli/src/cmd/forge/snapshot.rs:461]: no, but the regex should catch that, but will update

**Comment by transmissions11** [comment]
y'all simply dont miss

**Pull request merged**